### PR TITLE
Fixing py3 cover archiver + extending cluster allowlist IDs

### PR DESCRIPTION
**Pull request by mekarpeles**
Code fixes for #7197 

See: [README edits](https://github.com/internetarchive/openlibrary/pull/7230/files?short_path=a584955#diff-a5849555f8f6b6f9948f815963e945e4b66981c66eb57a1cf83941c133d5f696)

So far we've successfully archived `covers_0008_00` for all sizes and cleared up ~5GB.

### CODE REVIEW

File: openlibrary/coverstore/code.py [hunk 1]
```diff
@@ -267,6 +267,10 @@ def redirect(id):
             if value and self.is_cover_in_cluster(value):
                 url = zipview_url_from_id(int(value), size)
                 raise web.found(url)
+        elif key == 'test':
+            cover_id = 8000001
+
+            pass
```
Comment (cclauss) [inline openlibrary/coverstore/code.py:273]:
```suggestion
```

File: openlibrary/coverstore/archive.py [hunk 2]
```diff
@@ -85,39 +88,50 @@ def close(self):
...
-                    name=id + '-L.jpg', filename=cover.filename_l
+                    name="%010d-L.jpg" % cover.id, filename=cover.filename_l
                 ),
             }
 
-            for d in files.values():
-                d.path = d.filename and os.path.join(
-                    config.data_root, "localdisk", d.filename
+            for file_type in files:
+                f = files[file_type]
```
Comment (cclauss) [inline openlibrary/coverstore/archive.py:125]:
```suggestion
            for file_type, f in files.items():
```

File: openlibrary/coverstore/code.py [hunk 3]
```diff
@@ -276,15 +280,25 @@ def redirect(id):
         elif key != 'id':
             value = self.query(category, key, value)
 
-        if value and safeint(value) in config.blocked_covers:
+        if not value or (value and safeint(value) in config.blocked_covers):
             raise web.notfound()
 
         # redirect to archive.org cluster for large size and original images whenever possible
-        if value and (size == "L" or size == "") and self.is_cover_in_cluster(value):
+        if (size == "L" or size == "") and self.is_cover_in_cluster(value):
```
Comment (cclauss) [inline openlibrary/coverstore/code.py:287]:
```suggestion
        if size in ("L", "") and self.is_cover_in_cluster(value):
```

File: openlibrary/coverstore/README.md [hunk 4]
```diff
@@ -46,4 +48,28 @@ The item name itself (e.g. `coverd_0007`) is a combination of the prefix `covers
 
 **NB**: We identified **unarchived** covers (denoted with `archived=false` within the `covers` table) prior to `2014-11-29` but early tests suggest the archive process may not have been ironed out and standardized before this date, and so we decided to use the latest successful archival date to resume our archival efforts.  
 
-##
+## Archival Process
+
+**Recipe for moving one batch of 10k covers at a time into tars on archive.org.**
+
+1. On ol-covers0 docker container, rn archive.py on ~10k items to create a new partial of unarchived covers, starting at stable ID 8M (e.g. `covers_0008_01`)
```
Comment (cclauss) [inline openlibrary/coverstore/README.md:55]: Should this be `run`?
```suggestion
1. On ol-covers0 docker container, run archive.py on ~10k items to create a new partial of unarchived covers, starting at stable ID 8M (e.g. `covers_0008_01`)
```

**Comment by cclauss** [comment]
When I have git sync issues like the three Infogami files at the end of the diff, I do:
* `git submodule deinit -f . && git submodule update --init`

**Review by cdrini — APPROVED** [review]
Mek's run this on prod and it has saved 40GB! We should do some cleaning up of this file when we get the chance 😬

**Pull request merged**